src/modules/print/DailyPrintView.js

```diff
@@ -247,7 +247,7 @@ class DailyPrintView extends PrintView {
     this.data.bgRange[1] = _.max([this.data.bgRange[1], this.bgBounds.targetUpperBound]);
 
     // Calculate the maximum BG yScale value
-    this.bgScaleYLimit = _.min([this.data.bgRange[1], this.bgBounds.veryHighThreshold]);
+    this.bgScaleYLimit = _.max([this.data.bgRange[1], this.bgBounds.veryHighThreshold]);
```

Comment:
Just wanted to call out that this now differs from the functionality on the web daily view to make sure the difference in handling out-of-scale values is intentional.

On the web, we clamp the bg values at 400 (22.5 mmol/L).  Here, we're adjusting the scale to whatever the highest BG value is.

To this point, the instruction has always been to mimic the web view logic as closely as possible in the print views, so I wanted to make sure product is OK with the change.

Reply:
I should also note that the current functionality doesn't mimic the web view either, in that it doesn't clamp any values.

It may be that the change you've requested is indeed the proper approach.  I just want to be sure.